packages/commercetools/theme/nuxt.config.js

```diff
@@ -1,7 +1,7 @@
 import webpack from 'webpack';
-import { config } from './plugins/commercetools.js';
+import { config } from './config.js';
```

Comment:
why you moved commercetools config to the root?

Reply:
I'm not sure, to be honest... I'll move it to `plugins`